src/controllers/scripting/legacy/controllerscriptenginelegacy.cpp

```diff
@@ -201,6 +205,11 @@ bool ControllerScriptEngineLegacy::callInitFunction() {
             qCDebug(m_logger) << "Unhandled controller JS error is:"
                               << m_pJSEngine->catchError().toString();
         }
+
+        for (const auto& controller : m_mixxxController) {
+            controller->init();
```

Comment:
So this doesn't seem to be overriden from QML, sadly. I need to dig up a bit more…